foundational_security/elasticache.sp

```diff
@@ -0,0 +1,114 @@
+locals {
+  foundational_security_elasticache_common_tags = merge(local.foundational_security_common_tags, {
+    service = "AWS/ElastiCache"
+  })
+}
+
+benchmark "foundational_security_elasticache" {
+  title         = "ElasticCache"
+  documentation = file("./foundational_security/docs/foundational_security_elasticache.md")
+  children = [
+    control.foundational_security_elasticache_1,
+    control.foundational_security_elasticache_2,
+    control.foundational_security_elasticache_3,
+    control.foundational_security_elasticache_4,
+    control.foundational_security_elasticache_5,
+    control.foundational_security_elasticache_6,
+    control.foundational_security_elasticache_7
+  ]
+
+  tags = merge(local.foundational_security_elasticache_common_tags, {
+    type = "Benchmark"
```

Comment:
Should we include the service tag as well?

Reply:
We usually add the service tag in the `foundational_security_common_tags` and not seperately in `foundational_security_elasticache_common_tags` which is already added. Shall we add it here as well?